README.md

```diff
@@ -1 +1,85 @@
+  conda environment with the packages listed in this `environment.yml` file.
+
+**Note:**
+A complete list of possible configuration files that can be added to the repository and be used by repo2docker to build the Docker image, can be found in the [repo2docker docs](https://repo2docker.readthedocs.io/en/latest/config_files.html#configuration-files).
+
+## How to create and use a custom user image for your hub
+
+### 1. Use this template
+
+Create a new repository from `hub-user-image-template` repository, by clicking the *Use this template* button located at the top of this project's GitHub page.
+
+### 2. Hook the new repository to [quay.io](https://quay.io/)
+
+Follow all the instructions (except the last step), provided by the [repo2docker-action docs](https://github.com/jupyterhub/repo2docker-action#push-image-to-quayio) on how to allow the built image to be pushed to [quay.io](https://quay.io/).
+
+At end, you should have:
+
+* a quay.io repository of the form `quay.io/<quay-username>/<repository-name>`
+* two GitHub secrets **QUAY_USERNAME** (the user name of the `quay.io` robot account) and **QUAY_PASSWORD** (the password of the `quay.io` robot account) set on your newly created GitHub repository.
+
+  ![Secrets](images/secrets.png)
+
+### 3. Update repo2docker-action config
+
+Edit [build.yaml](https://github.com/2i2c-org/hub-user-image-template/blob/main/.github/workflows/build.yaml#L24-L25) and:
+
+* uncomment the `IMAGE_NAME` option
+* replace `<quay-username>/<repository-name>` with the info of the `quay.io` repository created at step 2
+
+### 4. Define the environment wanted
+
+* Modify [the environment.yml](https://github.com/2i2c-org/hub-user-image-template/blob/main/environment.yml) file and add all the packages you want installed in the conda environment. Note that repo2docker already installs [this list](https://github.com/jupyterhub/repo2docker/blob/HEAD/repo2docker/buildpacks/conda/environment.yml) of packages. More about what you can do with `environment.yml`, can be found in the [repo2docker docs](https://repo2docker.readthedocs.io/en/latest/config_files.html#environment-yml-install-a-conda-environment).
+
+* Commit the changes made to `environment.yml`.
+
+* Create a pull request with this commit, or push it dirrectly to the `main` branch.
+
+* If you merge the PR above or directly push the commit to the `main` branch, the GitHub Action will automatically build and push the container image. Wait for this action to finish.
+
+![Actions](images/action.png)
+
+### 5. Build and push the image of the repository
+
+Images generated by this action are automatically tagged with both latest and `<SHA>` corresponding to the relevant commit SHA on GitHub. Both tags are pushed to the Docker registry specified by the user. If an existing image with the *latest* tag already exists in your registry, this Action attempts to pull that image as a cache to reduce uncessary build steps.
```

Comment:
```suggestion
Images generated by this action are automatically tagged with both latest and `<SHA>` corresponding to the relevant commit SHA on GitHub. Both tags are pushed to the image registry specified by the user. If an existing image with the *latest* tag already exists in your registry, this Action attempts to pull that image as a cache to reduce uncessary build steps.
```

Since we're explicitly not using Docker Hub? I realise it's a Docker image, but I wondered if this terminology would be confusing for some who aren't as familiar with the differences.

Reply:
```suggestion
Images generated by this action are automatically tagged with both latest and `<SHA>` corresponding to the relevant commit SHA on GitHub. Both tags are pushed to the Docker registry specified by the user. If an existing image with the *latest* tag already exists in your registry, this Action attempts to pull that image as a cache to reduce unnecessary build steps.
```

Typo